components/routes/route-generator.vue

```diff
@@ -53,7 +53,7 @@ export default {
       this.validateInput()
 
       if (!this.showWarning) {
-        this.generate(this.target).then(() => {
+        this.generate([this.target]).then(() => {
```

Comment:
This is just temporary, a later PR will support front end changes that allow the creation of multiple links.